spec/request/in_progress_forms_request_spec.rb

```diff
@@ -241,6 +241,52 @@
         end
       end
     end
+
+    describe '#proxy_add' do
+      context 'user is missing birls and participant ids' do
+        context 'and is NOT completing form 526' do
+          let(:user) { build(:user_with_no_ids) }
+          let!(:in_progress_form) { FactoryBot.create(:in_progress_form, user_uuid: user.uuid) }
+
+          it 'call to add user to MVI is skipped' do
+            expect(in_progress_form.form_id).not_to eq('21-526EZ') # check test set up
+            expect_any_instance_of(Mvi).not_to receive(:mvi_add_person)
+            get v0_in_progress_form_url(in_progress_form.form_id), params: nil
+          end
+        end
+
+        context 'and is completing form 526' do
+          let(:user) { build(:user_with_no_ids) }
+          let!(:in_progress_form) { FactoryBot.create(:in_progress_form, user_uuid: user.uuid, form_id: '21-526EZ') }
+
+          it 'call is made to add user to MVI' do
+            VCR.use_cassette('mvi/add_person/add_person_success') do
+              VCR.use_cassette('mvi/find_candidate/orch_search_with_attributes') do
+                expect(in_progress_form.form_id).to eq('21-526EZ') # check test set up
+                # expect success to be achieved by calling mvi_add_person
+                expect_any_instance_of(Mvi).to receive(:mvi_add_person).once
```

Comment:
Nothing to add about the work here except I want people to know how annoying this test was to get working :)

Reply:
@SMLuthi were there any hard lessons learned worth sharing?

Reply:
Nothing that im not embarrassed about. It mostly came down to making `expect` calls AFTER making the `GET` request... essentially it was checking whether `:mvi_add_person` was being called after we had already made the request. So it was failing haha